ddtrace/contrib/flask/patch.py

```diff
@@ -426,7 +426,9 @@ def _wrap(rule, endpoint=None, view_func=None, **kwargs):
 def patched_add_url_rule(wrapped, instance, args, kwargs):
     """Wrapper for flask.app.Flask.add_url_rule to wrap all views attached to this app"""
 
-    def _wrap(rule, endpoint=None, view_func=None, **kwargs):
+    # certain flask-like frameworks, like flaskopenapi-3, may call add_url_rule with extra arguments
+    # adding a catch-all *args avoids number of arguments mismatch
+    def _wrap(rule, endpoint=None, view_func=None, *args, **kwargs):
```

Comment:
```suggestion
    def _wrap(rule, *args, endpoint=None, view_func=None, **kwargs):
```
I think this will be necessary to adhere to valid Python syntax

Reply:
ah yes fair, but I am getting new errors with that. I will work through those to see what's up!

Reply:
so I did some more digging (apologies, my Flask knowledge in general isn't super), but it looks like `flask-openapi3` passes the `provide_automatic_options` argument, which is read as a `kwarg` in base Flask, as an `arg`, so it's not picked up properly in our integration (since we only account for `endpoint` and `view_func` as keyword args, and assume everything else is passed in as an `kwarg`, not `arg`. I added it in as a variable in [c1d1b4b](https://github.com/DataDog/dd-trace-py/pull/9555/commits/c1d1b4b807503d0f5ed1bd8ed7e508c279700f88). AFAICT this is fine, and Flask tests are passing, and my local dummy app using `flask-openapi3` doesn't crash. lmk what you think, and i'll update the PR description and release note to be a bit more specific.